sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java

```diff
@@ -80,6 +81,25 @@ public EditCaseSteps(
                   "dateOfBirth"));
         });
 
+      When(
+              "I check the created from contact data is correctly displayed on Edit case page",
```

Comment:
I check case created from created contact is correctly displayed on Edit Case page